eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/parameters/InputParameters.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp.parameters;
+
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+
+
+public class InputParameters {
+
+    private String fasta;
+
+    private String assemblyReportUrl;
+
+    private String assemblyAccession;
+
+    private String assembly;
+
+    private int taxonomyAccession;
+
+    private int chunkSize;
+
+    private boolean forceRestart;
+
+    private int pageSize;
+
+    public JobParameters toJobParameters() {
+        throw new UnsupportedOperationException("implementation incomplete. make sure all the fields are used");
```

Comment:
Did you forget to implement it in the end or is it planned for another PR?

Reply:
I did this because we would forget to update it in case that at the very end of the development we add another parameter. a good moment to do this is when we add the application and the runner